conf/ledgersmb-apache-2.4.template

```diff
@@ -0,0 +1,60 @@
+# This file must be edited to substitute the current path with WORKING_DIR
+
+Alias /lsmbcss /var/lib/master/css/
```

Comment:
They are leftovers from 1.4 and before. I'll remove the alias.
>
> In conf/ledgersmb-apache-2.4.template
> <https://github.com/ehuelsmann/LedgerSMB/pull/240#discussion_r60783324>:
>
> > @@ -0,0 +1,60 @@
> > +# This file must be edited to substitute the current path with WORKING_DIR
> > +
> > +Alias /lsmbcss /var/lib/master/css/
>
> @ylavoie <https://github.com/ylavoie> make the |/var/lib/master/css|
> bit into something like |WORKING_DIR/css| ?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/ehuelsmann/LedgerSMB/pull/240/files/a6636f6dc8d0aad08544168b52862b70c41ba586#r60783324>
>